02-Appetisers/Barotropic_Streamfunction.ipynb

```diff
@@ -2,12 +2,20 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    rho0 = 1035 #kg/m3; since we removed <code>pint</code>, rho0 should be just treated as a plain number in SI units
</pre><p>Is the comment regarding <code>pint</code> still useful? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>